content/blog/2016-09-14-interview-with-digitalocean.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Interview with DigitalOcean
+created_at: 2016-09-14
+kind: article
+author_name: Brian Brazil
+---
+
+*Next in our series of interviews with users of Prometheus, DigitalOcean talks
+about how they use Promethus. Carlos Amedee also talked about [the social
+aspects of the rollout](https://www.youtube.com/watch?v=ieo3lGBHcy8) at PromCon
+2016.*
+
+## Can you tell us about yourself and what DigitalOcean does?
+
+My name is Ian Hansen and I work on the platform metrics team.
+[DigitalOcean](https://www.digitalocean.com/) provides simple cloud computing.
+To date, we’ve created 20 million Droplets (SSD cloud servers) across 13
+regions. We also recently released a new Block Storage product.
+
+![DigitalOcean logo](/assets/blog/2016-09-14/DO_Logo_Horizontal_Blue-3db19536.png)
+
+## What was your pre-Prometheus monitoring experience?
+
+Before Prometheus, we were running Graphite and OpenTSDB. Graphite was used for
+smaller-scale applications and OpenTSDB was used for collecting metrics from
+all of our physical servers via Collectd. Nagios would poll these databases to
+trigger alerts. We do still use Graphite but we no longer run OpenTSDB.
+
+## Why did you decide to look at Prometheus?
+
+I was frustrated with OpenTSDB because I was responsible for keeping the
+cluster online, but found it difficult to guard against metric storms.
+Sometimes a team would launch a new (very chatty) service that would impact the
+total capacity of the cluster and hurt my SLAs. 
+
+We are able to blacklist/whitelist new metrics coming in to OpenTSDB, but
+didn’t have a great way to guard against chatty services except for
+organizational process (which was hard to change/enforce). Other teams were
+frustrated with the query language and the visualization tools available at the
+time. I was chatting with Julius Volz about push vs poll metric systems and was
```

Comment:
Can switch to "pull" if that's more in-line with the verbage Prometheus is using. I've been using them mostly interchangeably, and no specific intent in choosing "poll" here.